CesiumGeospatial/include/CesiumGeospatial/GlobeRectangle.h

```diff
@@ -242,6 +242,19 @@ class CESIUMGEOSPATIAL_API GlobeRectangle final {
   GlobeRectangle computeUnion(const GlobeRectangle& other) const noexcept;
 
   /**
+   * @brief Splits this rectangle at the anti-meridian (180 degrees longitude),
+   * if necessary.
+   *
+   * If the rectangle does not cross the anti-meridian, the entire rectangle is
+   * returned in the `first` field of the pair and the `second` is std::nullopt.
+   * If it does cross the anti-meridian, this function returns two rectangles
+   * that touch but do not cross it. The larger of the two rectangles is
+   * returned in `first` and the smaller one is returned in `second`.
+   */
+  std::pair<GlobeRectangle, std::optional<GlobeRectangle>>
+  splitAtAntiMeridian() const noexcept;
```

Comment:
Feels like returning a fixed west->east ordering of the split rectangles would be preferable to just returning the largest one first... maybe this would be more useable for other use cases?

Not thinking of any though, so take with a grain of salt.

Reply:
I'm not sure, but for the one use-case so far (which is also in this PR), returning largest first is much preferred. I'm not inclined to change this without a clear reason. It's easy enough to sort them left to right if that's what you want to do.

Reply:
I have no other use cases in mind, so no worries here.